package.json

```diff
@@ -20,6 +20,7 @@
     "@angular/platform-browser-dynamic": "^17.0.0",
     "@angular/router": "^17.0.0",
     "dayjs": "^1.11.10",
+    "ip": "^2.0.1",
```

Comment:
```suggestion
```

Reply:
That was the suggested approach I found online, but I prefer your solution. Will apply.